dcos_tunnel/cli.py

```diff
@@ -519,9 +523,10 @@ def gen_hosts(ssh_client):
     return (mesos_hosts, dns_hosts)
 
 
-def container_cp(ssh_client, container_name, remote_filepath, local_file):
-    scom = 'docker exec {} bash -c "cat {}"'
-    scom = scom.format(container_name, remote_filepath)
+def container_cp(ssh_client, container_name, remote_filepath, local_file,
+                 docker_cmd='docker'):
```

Comment:
My thinking here was, however unlikely, someone might use your library in their code and call this method, so I shouldn't break backwards compatibility.